rcsb/workflow/wuw/PdbCsmImageWorkflow.py

```diff
@@ -73,33 +76,50 @@ def imagesGenJpgs(self, **kwargs: dict) -> None:
             outPath = os.path.join(jpgsOutDir, contentTypeDir)
             Path(outPath).mkdir(parents=True, exist_ok=True)
 
-            # runMolrender
-            cmd = [
-                jpgXvfbExecutable,
-                "-a",
-                "-s", f"-ac -screen 0 {jpgScreen}",
-                molrenderExe,
-                jpgRender,
-                bcifFilePath,
-                outPath,
-                "--height", jpgHeight,
-                "--width", jpgWidth,
-                "--format", jpgFormat,
-            ]
             bcifFileObj = Path(bcifFilePath)
             if bcifFileObj.is_file() and bcifFileObj.stat().st_size > 0:
-                try:
-                    result = subprocess.run(cmd, capture_output=True, text=True, check=True)
-                    logger.info(result.stdout)
-                except subprocess.CalledProcessError as e:
-                    logger.error("Error: %s", e)
-                    logger.error("Stderr: %s", e.stderr)
-                    raise
-
-                # check result
-                outJpgFile = os.path.join(outPath, name + checkFileAppend)
-                outFileObj = Path(outJpgFile)
-                if not (outFileObj.is_file() and outFileObj.stat().st_size > 0):
-                    raise ValueError(f"No image file: {outJpgFile}")
+                cmd = [
+                    jpgXvfbExecutable,
+                    "-a",
+                    "-s", f"-ac -screen 0 {jpgScreen}",
+                    molrenderExe,
+                    jpgRender,
+                    bcifFilePath,
+                    outPath,
+                    "--height", jpgHeight,
+                    "--width", jpgWidth,
+                    "--format", jpgFormat,
+                ]
+                # passing in args this way is due to a pickling issue with multiple subprocesses
+                args.append((cmd, outPath, name, checkFileAppend))
             else:
                 raise ValueError(f"Missing bcif file {bcifFilePath}")
+        if numProcs == 1:
+            self.run_command(args[0])
+        else:
+            with ProcessPoolExecutor(max_workers=int(numProcs)) as executor:
+                futures = [executor.submit(self.run_command, arg) for arg in args]
+                for future in as_completed(futures):
+                    try:
+                        future.result()
+                    except Exception as e:
+                        logger.error("Subprocess failed: %s", str(e))
+
+    def run_command(self, args):
```

Comment:
Oh right, in order to remove `self`, I think you need to make this a `staticmethod`, i.e.:
```python
    @staticmethod
    def run_command(args):
```
I'll leave it up to you whether you want to do that or just leave `self` here. I was just proposing this because of the limitations you mentioned earlier where the function needed exactly one argument.